apps/trading/lib/hooks/Rewards.graphql

```diff
@@ -75,6 +75,29 @@ query ActiveRewards(
           timestamp
           gameId
           kind {
+            ... on RecurringGovernanceTransfer {
+              startEpoch
```

Comment:
Would probably be better to make a fragment for these fields. It will help ensure consistency between RecurringGovernanceTransfer and RecurringTransfer and also create a type for us that can represent both

Reply:
I added some TransferKind fields. GraphQL won't let you make a fragment between 2 types unless you specify into the schema that those types are sharing the same properties.

Reply:
Ah yeah I see there are different types in the schema, even though the fields are (almost) entirely the same